File: en/io/rpcclient.rb
require "xmlrpc/client"

begin
  client = XMLRPC::Client.new("localhost", "/RPC2", 8081)
  resp = client.call("sum", 5, 3)
  puts "The result of the addition is #{resp['result']}"

  resp = client.call("divide", 11, 4)
  puts "The result of the division is #{resp['result']} and the remainder is #{resp['rest']}"
rescue => exception
  puts "ERROR: #{exception}"
end

File: en/io/rpcclient.py
# coding: utf-8
import xmlrpc.client

with xmlrpc.client.ServerProxy("http://localhost:8081/") as server:
    result = server.sum(5,3)
    print("The result of the addition is:", result["result"])

    result = server.divide(11,4)
    print("The result of the division is", result["result"], "and the remainder is", result["rest"])
